CardScan/Classes/CreditCardUtils.swift

```diff
@@ -1,6 +1,25 @@
 import Foundation
 
 public struct CreditCardUtils {
+    static let cvcLengthAmericanExpress = 4
+    static let cvcLength = 3
+    
+    static let maxPanLength = 16
```

Comment:
according to the iOS code some of the pans for visa are 14, do they not track that in the Android side?

Reply:
https://github.com/stripe/stripe-android/blob/7f78931e92c81257fe7443a3dd558273e9b45c44/stripe/src/main/java/com/stripe/android/CardUtils.kt#L107

they only check for diners club and amex, otherwise standard length

Reply:
ok, as long as we're consistent I'm fine with it